api/backend/ecs/service_manager.go

```diff
@@ -57,6 +57,21 @@ func (this *ECSServiceManager) ListServices() ([]*models.Service, error) {
 	return services, nil
 }
 
+func (this *ECSServiceManager) ListServiceNames() ([]id.L0ServiceID, error) {
+	serviceARNs, err := this.ECS.Helper_ListServices(id.PREFIX)
+	if err != nil {
+		return nil, err
+	}
+
+	servicesID := make([]id.L0ServiceID, len(serviceARNs))
+	for i, arn := range serviceARNs {
+		ecsServiceID := id.ServiceARNToECSServiceID(*arn)
```

Comment:
the `ecs` package should return `[]string` of service arns; no need to deal with pointer ugliness here